.bloodhound.yml

```diff
@@ -25,10 +25,16 @@ skip_types:
 additional_crds:
   # because the istio-chart is just referencing the istio-controller, so it doesn't contain the CRDs
   - https://raw.githubusercontent.com/istio/istio/1.9.1/manifests/charts/base/crds/crd-all.gen.yaml
+  # TODO: fix this in the correct way
```

Comment:
this doesn't seem bad 🤔. are you thinking that these should be handled in bloodhound directly?

Reply:
the thing is we never need this before but suddenly with this PR we need it...

Reply:
either in bloodhound, or k-apps repo(but not this bloodhound config)

Reply:
yeah its weird this isn't failing on main, looking at the code it seems like it should be

Reply:
this change to tell bloodhound about the CRDs actually seems like the right fix though

Reply:
I thought bloodhound is supposed to install CRD from the helm repo according to `HelmRelease` / `HelmChart`?

Reply:
i think it does if the CRD is in the same chart, but some of these failures are for custom resources where the CRD is installed in a separate chart. 

for example, project-grafana-loki complaining about the ObjectBucketClaim being an unknown type. but anyway, we can dive deeper when someone picks up the ticket you made!

Reply:
true. yeah i will also modify the comment to make this look more less opinionated

Reply:
done